atoma-proxy-service/src/handlers/stats.rs

```diff
@@ -275,3 +279,57 @@ async fn get_nodes_distribution(
             })?,
     ))
 }
+
+/// OpenAPI documentation for the get_graphs endpoint.
+///
+/// This struct is used to generate OpenAPI documentation for the get_graphs
+/// endpoint. It uses the `utoipa` crate's derive macro to automatically generate
+/// the OpenAPI specification from the code.
+#[derive(OpenApi)]
+#[openapi(paths(get_graphs))]
+pub struct GetGraphs;
+
+/// Get graphs.
+///
+/// # Arguments
+///
+/// * `proxy_service_state` - The shared state containing the state manager
+///
+/// # Returns
+///
+/// * `Result<Json<Value>>` - A JSON response containing a list of graphs with data
+///   - `Ok(Json<Value>)` - Successfully retrieved graphs
+///   - `Err(StatusCode::INTERNAL_SERVER_ERROR)` - Failed to retrieve graphs from state manager
+#[utoipa::path(
+    get,
+    path = "",
+    responses(
+        (status = OK, description = "Retrieves all graphs", body = Value),
+        (status = INTERNAL_SERVER_ERROR, description = "Failed to get graphs")
+    )
+)]
+#[instrument(level = "trace", skip_all)]
+async fn get_graphs(
```

Comment:
Don't we need some form of authorization here ? Or this being widely available is fine ?

Reply:
If you don't want to hide the graphs on the dashboard behind login, then no.